### PR TITLE
fix: improve performance in dataset loading

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Checklist

- [x] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Comment by guilhem-barthes** [comment]
/e2e --tests sdk,frontend,substrafl --benchmarks mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: backend/api/models/datamanager.py [hunk 1]
```diff
@@ -24,11 +24,7 @@ class DataManager(models.Model, AssetPermissionMixin):
     owner = models.CharField(max_length=100)
     creation_date = models.DateTimeField()
     metadata = models.JSONField()
-    channel = models.CharField(max_length=100)
-
-    def get_data_samples(self):
```
Comment (SdgJlbl) [inline backend/api/models/datamanager.py:29]: this function is no longer used?

Comment (guilhem-barthes) [reply backend/api/models/datamanager.py:29]: THis function was only used in the serializer

**Comment by guilhem-barthes** [comment]
/e2e --tests sdk,frontend,substrafl --benchmarks mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by ThibaultFy — APPROVED** [review]
LGTM

**Pull request merged**